posts/2021-09-29-jcon-conference.adoc

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "MicroProfile, Jkarta EE and Open Liberty at JCON-Online 2021"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/GraceJansen
+author_github: https://github.com/GraceJansen
+seo-title: MicroProfile, Jkarta EE and Open Liberty at JCON-Online 2021 - OpenLiberty.io
+seo-description: A helpful summary of the JCON-Online 2021 workshops and talks on MicroProfile, Jakarta EE and Open Liberty.
+blog_description: "A helpful summary of the JCON-Online 2021 workshops and talks on MicroProfile, Jakarta EE and Open Liberty."
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+---
+= MicroProfile, Jakarta EE and Open Liberty at JCON-Online 2021
+Grace Jansen <https://github.com/GraceJansen>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+ Despite the global pandemic and general chaos of this year, many amazing developer events and conferences are still taking place virtually, bringing the developer community together once more. One such event that will use a digital format to bring together the international Java community is JCON-Online. The great part is, it's free for all JUG (Java User Group) members!
+
+Several members of the Open Liberty team will be joining https://jcon.one/[JCON-Online 2021] on 5-8 October. If you're also going to be there, deciding which of the great talks to see at JCON looks like it could be a challenge. So we've put together a helpful summary of the sessions on https://microprofile.io/[MicroProfile], https://jakarta.ee/[Jakarta EE] and https://openliberty.io/about/[Open Liberty].
+
+image::/img/blog/JCON_Conf_Logo.png[JCON Online Conference Logo 2021,width=70%,align="center"]
+
+== Talks at JCON 2021
+
+A selection of MicroProfile and Open Liberty talks on Tuesday, Wednesday and Thursday:
+
+*Tuesday 5th*
```

Comment:
The list items  and descriptions need some adjustment. For accessibility, avoid using bold to substitute for a heading style. Throughout the post, rather than putting the dates as *bold* , you can either make them L3 headings or use the [description list syntax](https://docs.antora.org/antora/2.3/asciidoc/description-lists/). Just add a double colon after the date, like 

Tuesday 5th:: 
Live coding: Configuring your cloud-native applications with MicroProfile Config
Java Champion Emily Jiang’s live demo session will showcase various ways to configure your microservices via the amazing features introduced in MicroProfile Config 2.0, including config profile, config properties, variable replacement and many more.